src/operators.jl

```diff
@@ -166,3 +166,25 @@ end
 # dispatch for g[[1,2,3]], g[1:3], g[Set([1,2,3])]
 # these are the only allowed dispatches, everything else is slow
 getindex(g::SimpleGraph, iter) = induced_subgraph(g, iter)
+
+"""Provides multiplication of a graph `g` by a vector `v` such that spectral
+graph functions in [GraphMatrices.jl](https://github.com/jpfairbanks/GraphMatrices.jl) can utilize LightGraphs natively.
```

Comment:
Just FYI I stripped out a definition, I think maybe it was ==, from the operators docs, since the docstring is nonsensical (atm). That and this should be added back in the future. MichaelHatherly had a neat trick for this...